packages/babel-plugin-minify-numeric-literals/src/index.js

```diff
@@ -1,24 +1,26 @@
 "use strict";
 
-module.exports = function({ types: t }) {
+module.exports = function() {
   return {
     name: "minify-numeric-literals",
     visitor: {
       NumericLiteral(path) {
         if (!path.node.extra) return;
 
-        const exponential = path.node.value
-          .toExponential()
-          .replace(/\+/g, "")
-          .replace(/e0/, "");
+        const normal = path.node.value.toString().replace(/^0\./, ".");
```

Comment:
I am not sure why I have used parseFloat instead of this, remember there was a missed optimisation.